nuxeo-connect-client/src/main/java/org/nuxeo/connect/connector/AbstractConnectConnector.java

```diff
@@ -169,10 +175,9 @@ public DownloadingPackage getDownload(String id) throws ConnectServerError {
             log.error(e);
         }
         String url = getBaseUrl() + GET_DOWNLOAD_SUFFIX + "/" + id;
-        ConnectServerResponse response = null;
+        ConnectServerResponse response = execCall(url);
```

Comment:
It's clearer to have the `try` _immediately_ after the call allocating a resource that needs to be released in a `finally` block. So move the `PackageDescriptor pkg = null` one line up.